### DISCUSSION TITLE
How to set Select Option Element border radius to zero?

### DISCUSSION THREAD

**Discussion by skfw-dev** [scalar/scalar#3588] [Q&A] [answered]
I am tried customize theme and border radius to zero like sharp curve, working well with all elements, except select option scalar client, my code like this

```js
cssStyleSheet.addRule(':root', '--scalar-radius: 0px !important; --scalar-radius-lg: 0px !important; --scalar-radius-xl: 0px !important;');
cssStyleSheet.addRule(':hover', 'border-radius: 0px !important;');
cssStyleSheet.addRule(':focus', 'border-radius: 0px !important;');
cssStyleSheet.addRule('::before', 'border-radius: 0px !important;');
cssStyleSheet.addRule('::after', 'border-radius: 0px !important;');
cssStyleSheet.addRule('select, optgroup, option', 'border-radius: 0px !important;'); // not work
```

I am using javascript to inject css in style element or existing style element in head, Plz help me :)

**Comment by antlio** [comment]
hey @john-shika, thanks for the feedback!

by select option scalar client do you mean for the dropdown as below for instance?

<img width="765" alt="image" src="https://github.com/user-attachments/assets/a13c7467-3d8d-4a60-a21d-d9c306ef3fa5">

**Reply by skfw-dev** [reply]
@antlio thx for the response my problem, but idk this is dropdown or not, but the option contain select the client like wget curl etc, their keep rounded :)

**Comment by hwkr** [comment, marked as answer]
Hi @john-shika! The client dropdown uses a native browser `<select>` element so it's rendering is handled by the browser and how it looks depends on the browser and operating system your using.

We may look at converting it to a Javascript based dropdown in the future so it can be themed :)

**Reply by skfw-dev** [reply]
@hwkr sound great!
